packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2130,335 @@ Specific property types support a subset of [JSON Schema validations](https://to
+| properties  | Group object properties                                                                  | [Group object](/docs/reference/api/groups/#group-object) property set | FALSE    | FALSE  | TRUE     |             |
+
+### Group Profile subschemas
+
+The [Profile object](/docs/reference/api/groups/#profile-object) for a Group is defined by a composite schema of base and custom properties using a JSON path to reference subschemas. The `#base` properties are defined and versioned by Okta while `#custom` properties are extensible.
+
+- [Group Profile base subschema](#group-profile-base-subschema)
+- [Group Profile custom subschema](#group-profile-custom-subschema)
+
+Custom property names for the [Profile object](/docs/reference/api/groups/#profile-object) must be unique and can't conflict with a property name defined in the `#base` subschema.
+
+```json
+{
+  "definitions": {
+    "base": {
+      "id": "#base",
+      "properties": {
+        "description": {
+          "description": "Description",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 1024,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "scope": "NONE",
+          "title": "Description",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 255,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "Name",
+          "type": "string"
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "type": "object"
+    },
+    "custom": {
+      "id": "#custom",
+      "properties": {
+        "exampleCustomProperty": {
+          "description": "exampleCustomProperty",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 20,
+          "minLength": 1,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "exampleCustomProperty",
+          "type": "string",
+          "unique": "UNIQUE_VALIDATED"
+        }
+      },
+      "required": [
+        "exampleCustomProperty"
+      ],
+      "type": "object"
+    }
+  }
+}
+```
+
+#### Group Profile base subschema
+
+All Okta-defined Profile properties are defined in a Profile subschema with the resolution scope `#base`. These properties can't be removed or edited.  Attempting to do so anyway will have no effect.
```

Comment:
Remove extra space after 2nd period.

Blend last two sentences into one: These properties can't be removed or edited, regardless of any attempt to do so.